android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/api/DocumentAPITest.java

```diff
@@ -137,22 +137,24 @@ public void run() {
     // ### Blobs
     @Test
     public void testBlobs() {
+        MutableDocument newTask = new MutableDocument();
+
         // # tag::blob[]
-        InputStream is = getAsset("attachment.png");
+        InputStream is = getAsset("avatar.jpg");
```

Comment:
`avatar.jpg` is not in `assets` folder. Could you please add the image file in the assets folder to avoid test failure.

Reply:
Done.